SADL/AM-CDM-part.sadl

```diff
@@ -0,0 +1,31 @@
+uri "http://kdl.ge.com/AM-CDM-part" alias AM-CDM-part version "$Revision:$ Last modified on   $Date:$".
+
+import "http://kdl.ge.com/AM-CDM-base".
+
+Part (note "Instances of part designs in the as-built state.") is a MaterialStock,
+	described by name (note 'Human readable name which designates this part.') with a single value of type string,
```

Comment:
Should element names include the prefex of the class they are in?  e.g. partName instead of name.  I think this makes individual elements more explicit.  We go back and forth doing this and not doing this throughout the SADL files, but I think we should try to be consistent.

Reply:
Identical comment to Alex Kuan's. Resolved to consolidate comments